helpers/whitelabel/branded_links.go

```diff
@@ -0,0 +1,141 @@
+package whitelabel
```

Comment:
We have updated the verbiage used to describe a "whitelabel". Please update accordingly. See [this PR](https://github.com/sendgrid/sendgrid-go/pull/329) for an example.